src/libraries/Common/src/System/Security/Cryptography/ECDsaCng.ImportExport.cs

```diff
@@ -87,10 +88,17 @@ public override void ImportParameters(ECParameters parameters)
         /// <returns>The key and explicit curve parameters used by the ECC object.</returns>
         public override ECParameters ExportExplicitParameters(bool includePrivateParameters)
         {
-            byte[] blob = ExportFullKeyBlob(includePrivateParameters);
-            ECParameters ecparams = default;
-            ECCng.ExportPrimeCurveParameters(ref ecparams, blob, includePrivateParameters);
-            return ecparams;
+            if (includePrivateParameters)
+            {
+                return ExportPrivateExplicitParameters();
+            }
+            else
+            {
+                byte[] blob = ExportFullKeyBlob(includePrivateParameters: false);
+                ECParameters ecparams = default;
+                ECCng.ExportPrimeCurveParameters(ref ecparams, blob, includePrivateParameters: false);
+                return ecparams;
+            }
```

Comment:
What about ECDH?  Can this easily be lifted into ECCng to avoid duplication?

And, sure, "what about DSA?"